spartan/aztec-network/templates/boot-node.yaml

```diff
@@ -62,8 +62,9 @@ spec:
               cp /scripts/deploy-l1-contracts.sh /tmp/deploy-l1-contracts.sh
               chmod +x /tmp/deploy-l1-contracts.sh
               source /shared/config/service-addresses
-              HASH=$(echo "$RELEASE_NAME-$RELEASE_REVISION" | sha256sum | head -c 8) # first 4 hex characters = 4 bytes
-              SALT=$((16#$HASH)) # convert to decimal
+
+              SALT=${SALT:-$RANDOM}
```

Comment:
Just a catch all for any deployment that don't use the workflow file (like local kind clusters)